pkg/cloud-provider/securitygroup/securitygroup.go

```diff
@@ -170,6 +177,18 @@ func (c *CloudResourceID) String() string {
 	return c.Name + "/" + c.Vpc
 }
 
+type CloudRuleDescription struct {
+	Name           string
+	Namespace      string
```

Comment:
Should we say ANPName and ANPNamespace, or mention ANP in a comment?

Reply:
Done